arex-schedule-web-api/src/main/java/com/arextest/schedule/beans/ExecutorServiceConfiguration.java

```diff
@@ -17,69 +19,62 @@
+            new ThreadPoolExecutor.CallerRunsPolicy());
     }
 
     /**
      * This bean should be overridden according to the actual implementation of compare service
      */
     @Bean
     public ExecutorService sendExecutorService() {
-        ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("replay-send-%d")
-                .setDaemon(true)
-                .setUncaughtExceptionHandler(this)
-                .build();
+        ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("replay-send-%d").setDaemon(true)
+            .setUncaughtExceptionHandler(this).build();
         return new ThreadPoolExecutor(calculateIOPoolSize(), calculateIOPoolSize(), KEEP_ALIVE_TIME,
-                TimeUnit.MILLISECONDS,
-                new LinkedBlockingQueue<>(SEND_QUEUE_MAX_CAPACITY_SIZE),
-                threadFactory,
-                new ThreadPoolExecutor.CallerRunsPolicy());
+            TimeUnit.MILLISECONDS, new LinkedBlockingQueue<>(SEND_QUEUE_MAX_CAPACITY_SIZE), threadFactory,
+            new ThreadPoolExecutor.CallerRunsPolicy());
     }
 
-
     @Bean
     public ExecutorService compareExecutorService() {
-        ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("replay-compare-%d")
-                .setDaemon(true)
-                .setUncaughtExceptionHandler(this)
-                .build();
-        return new ThreadPoolExecutor(CORE_POOL_SIZE, CORE_POOL_SIZE, KEEP_ALIVE_TIME,
-                TimeUnit.MILLISECONDS,
-                new LinkedBlockingQueue<>(COMPARE_QUEUE_MAX_CAPACITY_SIZE),
-                threadFactory,
-                new ThreadPoolExecutor.CallerRunsPolicy());
+        ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("replay-compare-%d").setDaemon(true)
+            .setUncaughtExceptionHandler(this).build();
+        return new ThreadPoolExecutor(CORE_POOL_SIZE, CORE_POOL_SIZE, KEEP_ALIVE_TIME, TimeUnit.MILLISECONDS,
+            new LinkedBlockingQueue<>(COMPARE_QUEUE_MAX_CAPACITY_SIZE), threadFactory,
+            new ThreadPoolExecutor.CallerRunsPolicy());
 
     }
 
     @Bean
     public ExecutorService rerunPrepareExecutorService() {
         ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("replay-rerun-prepare-%d")
-            .setDaemon(true)
-            .setUncaughtExceptionHandler(this)
-            .build();
-        return new ThreadPoolExecutor(CORE_POOL_SIZE, CORE_POOL_SIZE, KEEP_ALIVE_TIME,
-            TimeUnit.MILLISECONDS,
-            new LinkedBlockingQueue<>(COMPARE_QUEUE_MAX_CAPACITY_SIZE),
-            threadFactory,
+            .setDaemon(true).setUncaughtExceptionHandler(this).build();
+        return new ThreadPoolExecutor(CORE_POOL_SIZE, CORE_POOL_SIZE, KEEP_ALIVE_TIME, TimeUnit.MILLISECONDS,
+            new LinkedBlockingQueue<>(COMPARE_QUEUE_MAX_CAPACITY_SIZE), threadFactory,
+            new ThreadPoolExecutor.CallerRunsPolicy());
+    }
```

Comment:
Rollback structural diffs.

Reply:
This time, the code format style is ignored. In future submissions, 
[https://github.com/arextest/arex-api/blob/main/intellij-java-google-style.xml]() will be used uniformly.